docker-compose.test.yml

```diff
@@ -51,8 +51,10 @@ services:
     environment:
       - EDGE_PORT=4566
       - SERVICES=s3
+    volumes:
+      - ./scripts/localstack:/etc/localstack/init/ready.d
```

Comment:
For speed & RAM, might be worth removing this and copying it in the actual docker image instead.